pages/wizard/product_selection.py

```diff
@@ -13,6 +13,7 @@ class SelectProductsPage(QCIPage):
                                     "//span[@id='is_cloudforms']/div/input")
     _openshift_checkbox_locator = (By.XPATH,
                                    "//span[@id='is_openshift']/div/input")
+    _requirement_block_locator = (By.XPATH, "//div[@class='req-block']")
```

Comment:
Could you add a comment explaining what the requirement block is.   I think I know, but a comment would be helpful.